mixer/test/client/env/ports.go

```diff
@@ -59,6 +59,7 @@ const (
 	PilotPluginTLSTest
 	PilotMCPTest
 
+	RbacGlobalPermissiveTest
```

Comment:
Please move this up to line 61 and leave this line blank.

Reply:
done